### PR TITLE
User Guide front page update 

### LINKED ISSUE #49: [User Guide] Update first page
Update introduction, quick start and features over-view of UG

### PR DESCRIPTION
**Pull request by ElginL**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -3,37 +3,57 @@ layout: page
...
+1. Copy the file to the folder you want to use as the _home folder_ for your JeeqTracker.
 
 1. Double-click the file to start the app. The GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
    ![Ui](images/Ui.png)
 
 1. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
    Some example commands you can try:
 
-   * **`list`** : Lists all contacts.
+   * **`list`** : Lists all companies
```
Comment (notle1706) [inline docs/UserGuide.md]: I think you forgot a fullstop here.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -3,37 +3,57 @@ layout: page
...
 
-   * **`add`**`n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01` : Adds a contact named `John Doe` to the Address Book.
+   * **`add`**`coy/MacDonalds n/James hp/98765432 e/jamesho@example.com loc/West Coast Park` : Adds a point of contact named `James` to the Company `MacDonalds`
```
Comment (notle1706) [inline docs/UserGuide.md]: Missing fullstop?

File: docs/UserGuide.md [hunk 3]
```diff
@@ -3,37 +3,57 @@ layout: page
...
 
-   * **`delete`**`3` : Deletes the 3rd company shown in the current list.
+   * **`delete`**`coy/MacDonalds` : Deletes MacDonalds (Company) together with all their contacts in the current list
```
Comment (notle1706) [inline docs/UserGuide.md]: I think there're a few sentences that you didn't include the fullstop.

**Review by notle1706 — CHANGES_REQUESTED** [review]
Missing a few fullstops. Otherwise LGTM!

**Review by jicsontoh — APPROVED** [review]
LGTM!!

**Review by notle1706 — APPROVED** [review]
LGTM

**Pull request merged**